src/app/organizations/registerOrganization/requireAccountsModal/require-accounts-modal.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, OnInit } from '@angular/core';
+import { TokenQuery } from '../../../shared/state/token.query';
+import { MatDialog } from '@angular/material/dialog';
+import { TokenUser } from '../../../shared/swagger';
+import { RegisterOrganizationComponent } from '../register-organization.component';
+import { TagEditorMode } from '../../../shared/enum/tagEditorMode.enum';
+import { takeUntil } from 'rxjs/operators';
+import { Subject } from 'rxjs';
+
+@Component({
+  selector: 'app-require-accounts-modal',
+  templateUrl: './require-accounts-modal.component.html',
+})
+export class RequireAccountsModalComponent implements OnInit {
+  public numLinkedAccounts: number = 0;
+  private ngUnsubscribe: Subject<{}> = new Subject();
```

Comment:
this unsubscribe needs to be linked to the onDestroy method or something else in order for it to unsubscribe. my recommendation is still to pipe map so...

```
numLinkedAccounts$ = this.tokenQuery.tokens$.pipe(map(tokens => tokens.length - 1));
```

then you would have to async pipe in the html like:

`numLinkedAccounts$ | async`